addon/index.js

```diff
@@ -342,6 +358,11 @@ exports.main = function(options) {
 
   if (reason === 'install') {
     openOnboardingTab();
+  } else if (
+    store.showSharePromptAt && Date.now() > store.showSharePromptAt ||
+    typeof store.showSharePromptAt === 'undefined' && hasExperimentInstalled()
```

Comment:
Trying to work though the logic here to make sure I've got it right... Looks like there are 2 cases when the share-solicitation tab gets shown:

1. Existing Test Pilot users with at least one experiment installed will see a sharing-solicitation immediately on their next browser restart.

2. Future Test Pilot users on their first experiment enable will get a sharing-solicitation scheduled for the next browser restart after at least 3 days have passed.

In either of these cases, `store.showSharePromptAt` gets set to `false` after showing the tab. That, in turn, records that the tab has been shown and no future showings will be scheduled on experiment enables.

As long as I've got all that right, I'd say case 1 sounds okay if we assume existing Test Pilot users have had an experiment around for more than 3 days, so this is basically playing catch-up.